src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(String color) {
+        super(color);
+    }
+
+    Circle(String color, double radius) {
+        super(color);
+        this.setName(Name.CIRCLE.name());
```

Comment:
The previous comment meant that in super class `Figure` you need change `name` field from type `String` to type `enum FigureName`